documentation/staging/1.4/content/release/_index.md

```diff
@@ -27,9 +27,17 @@ pre = "<b> </b>"
 - #176: Resolved an issue where the `Update Ingress Routes` action was always passing the `SSL pass-through` argument--even when the user did not request it.
 - #182: Resolved Verrazzano 1.4.0+ installation issue caused by a change in the published operator file name. 
 - #186: Resolved Verrazzano-related issue where Ingress Trait rules were not including the hosts or destination-related fields.
+- #187: Resolved an issue where the operator domain variable override config map was not being created properly.
+- #187: Resolved an issue with the `apiVersion` of the Verrazzano component- and application-related being incorrect.
```

Comment:
This seems like it's missing a word, "the Verrazzano component- and application-related (what) being incorrect.